Benchmark-Models/Rahman_MBS2016/measurementData_Rahman_MBS2016.tsv

```diff
@@ -0,0 +1,24 @@
+observableId	preequilibrationConditionId	simulationConditionId	measurement	time	observableParameters	noiseParameters	datasetId
+observable_prevalence		simCondition	0.3	0		1	model1_data1_prevalence
+observable_prevalence		simCondition	0.6	1		1	model1_data1_prevalence
+observable_prevalence		simCondition	1.1	2		1	model1_data1_prevalence
+observable_prevalence		simCondition	2.0	3		1	model1_data1_prevalence
+observable_prevalence		simCondition	3.3	4		1	model1_data1_prevalence
+observable_prevalence		simCondition	5.0	5		1	model1_data1_prevalence
+observable_prevalence		simCondition	7.0	6		1	model1_data1_prevalence
+observable_prevalence		simCondition	9.2	7		1	model1_data1_prevalence
+observable_prevalence		simCondition	11.3	8		1	model1_data1_prevalence
+observable_prevalence		simCondition	13.2	9		1	model1_data1_prevalence
+observable_prevalence		simCondition	14.9	10		1	model1_data1_prevalence
+observable_prevalence		simCondition	16.2	11		1	model1_data1_prevalence
+observable_prevalence		simCondition	17.1	12		1	model1_data1_prevalence
+observable_prevalence		simCondition	17.8	13		1	model1_data1_prevalence
+observable_prevalence		simCondition	18.3	14		1	model1_data1_prevalence
+observable_prevalence		simCondition	18.5	15		1	model1_data1_prevalence
+observable_prevalence		simCondition	18.7	16		1	model1_data1_prevalence
+observable_prevalence		simCondition	18.8	17		1	model1_data1_prevalence
+observable_prevalence		simCondition	18.9	18		1	model1_data1_prevalence
+observable_prevalence		simCondition	18.9	19		1	model1_data1_prevalence
+observable_prevalence		simCondition	18.9	20		1	model1_data1_prevalence
+observable_prevalence		simCondition	18.8	21		1	model1_data1_prevalence
+observable_prevalence		simCondition	18.9	22		1	model1_data1_prevalence
```

Comment:
Why is the noiseParameter 1? Did they not consider measurement noise in the publication?

Reply:
Yes I was advised that there is an implied assumption that the standard deviation is 1, given the error function that is defined in Section 4.3 (5th page) of the paper: https://www.uwo.ca/apmaths/faculty/zou/repr/mbs16.pdf

Let me know if you think that what I've done is not reasonable.

Reply:
Ok. Seems fine to me